<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>641</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>641</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1100</width>
      <height>641</height>
     </rect>
    </property>
    <widget class="QWidget" name="loginPage">
     <property name="styleSheet">
      <string notr="true">#loginPage {border-image: url(:/image/accesso.png);}</string>
     </property>
     <widget class="QPushButton" name="LoginAdmin">
      <property name="geometry">
       <rect>
<<<<<<< HEAD
        <x>510</x>
        <y>400</y>
=======
        <x>860</x>
        <y>590</y>
>>>>>>> 48fb053b3b8520ca8be1496805e1cce4227239e1
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#LoginAdmin{color: rgb(255, 255, 255); 
background-color: rgb(255, 0, 0); 
font-size: 16px; 
border-radius: 20px; 
border: 3px solid #B30000;
}
#LoginAdmin:hover{
background-color: #B30000;
}
#LoginAdmin:pressed{
background-color: #800000;
}</string>
      </property>
      <property name="text">
       <string>Accedi (Admin Mode)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoginButton">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>370</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#LoginButton{color: rgb(255, 255, 255); 
background-color: rgb(20, 115, 255); 
font-size: 16px; 
border-radius: 20px; 
border: 3px solid rgb(28, 110, 164);
}
#LoginButton:hover{
background-color: rgb(28, 110, 164);
}
#LoginButton:pressed{
background-color: rgb(20, 78, 117);
}</string>
      </property>
      <property name="text">
       <string>Accedi</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoginPasswordForm">
      <property name="geometry">
       <rect>
        <x>784</x>
        <y>320</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border-radius: 15px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 18px;
</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RegisterButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#RegisterButton {
color: #FFFFFF;
background-color: Transparent;
background-repeat:no-repeat;
}

#RegisterButton:hover {
color: #FFFFFF;
text-decoration: underline;
}</string>
      </property>
      <property name="text">
       <string>REGISTRATI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ForgotPasswordButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>380</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#ForgotPasswordButton {
color: #505050;
background-color: Transparent;
background-repeat:no-repeat;
}

#ForgotPasswordButton:hover {
color: #505050;
background-color: Transparent;
background-repeat:no-repeat;
text-decoration: underline;
}</string>
      </property>
      <property name="text">
       <string>Password dimenticata?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoginUsernameForm">
      <property name="geometry">
       <rect>
        <x>784</x>
        <y>260</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border-radius: 15px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 18px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>604</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #EAFF00;</string>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="regPage">
     <property name="styleSheet">
      <string notr="true">#regPage {border-image: url(:/image/registrazione.png);}</string>
     </property>
     <widget class="QLineEdit" name="RegUsernameForm">
      <property name="geometry">
       <rect>
        <x>784</x>
        <y>260</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border: transparent;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 18px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RegMailForm">
      <property name="geometry">
       <rect>
        <x>784</x>
        <y>377</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border: transparent;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 18px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SignUpButton">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>430</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#SignUpButton{color: rgb(255, 255, 255); 
background-color: rgb(20, 115, 255); 
font-size: 16px; 
border-radius: 20px; 
border: 3px solid rgb(28, 110, 164);
}
#SignUpButton:hover{
background-color: rgb(28, 110, 164);
}
#SignUpButton:pressed{
background-color: rgb(20, 78, 117);
}</string>
      </property>
      <property name="text">
       <string>Registrati</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AccediButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>40</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#AccediButton {
color: #FFFFFF;
background-color: Transparent;
background-repeat:no-repeat;
}

#AccediButton:hover {
text-decoration: underline;
}</string>
      </property>
      <property name="text">
       <string>ACCEDI</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RegPasswordForm">
      <property name="geometry">
       <rect>
        <x>784</x>
        <y>320</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border: transparent;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 18px;
</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menuPage">
     <property name="styleSheet">
      <string notr="true">#menuPage {border-image: url(:/image/start.png);}</string>
     </property>
     <widget class="QPushButton" name="LogoutButton">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>590</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777199</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#LogoutButton{
	color: rgb(255, 255, 255); 
	background-color: rgb(20, 115, 255); 
	font-size: 16px; 
	border-radius: 20px; 
	border: 3px solid rgb(28, 110, 164);
}
#LogoutButton:hover{
	background-color: rgb(28, 110, 164);
}
#LogoutButton:pressed{
	background-color: rgb(20, 78, 117);
}</string>
      </property>
      <property name="text">
       <string>Disconnettiti</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openEditor">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>350</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#openEditor{
	text-align: left;
	color: #0000FF;
	font-size: 20px;
	background-color:transparent;
	border: transparent;
}
#openEditor:hover{
	text-decoration: underline;
}
</string>
      </property>
      <property name="text">
       <string>Apri File</string>
      </property>
      <property name="icon">
       <iconset resource="Risorse.qrc">
        <normaloff>:/image/397435-200.png</normaloff>:/image/397435-200.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>13</width>
        <height>13</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Username">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>250</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#Username{
	text-align: left;
	color: #0000FF;
	font-size: 20px;
	background-color:transparent;
	border: transparent;
}
#Username:hover{
	text-decoration: underline;
}
</string>
      </property>
      <property name="text">
       <string>UsernameGoesHere</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1068</x>
      <y>2</y>
      <width>30</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
background-repeat:no-repeat;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #EAFF00;</string>
    </property>
    <property name="text">
     <string>QuestoTestoVieneSostituitoDaVersione</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>RegUsernameForm</tabstop>
  <tabstop>RegPasswordForm</tabstop>
  <tabstop>RegMailForm</tabstop>
  <tabstop>LoginUsernameForm</tabstop>
  <tabstop>LoginPasswordForm</tabstop>
  <tabstop>LoginButton</tabstop>
  <tabstop>ForgotPasswordButton</tabstop>
  <tabstop>RegisterButton</tabstop>
  <tabstop>LogoutButton</tabstop>
  <tabstop>SignUpButton</tabstop>
  <tabstop>AccediButton</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>LoginAdmin</tabstop>
  <tabstop>Username</tabstop>
  <tabstop>openEditor</tabstop>
 </tabstops>
 <resources>
  <include location="Risorse.qrc"/>
 </resources>
 <connections/>
</ui>
